src/translations/fr/legal-notice.mdx

```diff
@@ -1,18 +1,41 @@
-# Mentions légales
+# Mentions légales de Open Terms Archive
 
-## Rédacteur en chef
+## Editeur de la Plateforme
 
-Bureau de l’ambassadeur pour les affaires numériques  
-Ministère de l’Europe et des affaires étrangères  
-37 Quai d’Orsay, 75015 Paris, France
+La Plateforme Open Terms Archive est éditée par le Bureau de l’ambassadeur pour les affaires numériques situé :
 
-## Directeur de la rédaction
+_Ministry of Europe and Foreign Affairs_
+
+_37 Quai d'Orsay, 75015 Paris, France_
+
+_contact - at - opentermsarchive.org_
+
+## Directeur de la publication
 
 Henri Verdier, ambassadeur pour le numérique.
 
-## Hébergeur
+## Hébergement de la Plateforme
+
+Ce site est hébergé en propre par OVH :
+
+_<https://www.scalingo.com>_
+
+_15 avenue du Rhin, 67100 Strasbourg, France_
+
+## Accessibilité
+
+La conformité aux normes d’accessibilité numérique est un objectif ultérieur mais nous tâchons de rendre ce site accessible à toutes et à tous.
+
+## Signaler un dysfonctionnement
+
+Si vous rencontrez un défaut d’accessibilité vous empêchant d’accéder à un contenu ou une fonctionnalité du site, merci de nous en faire part.
+Si vous n’obtenez pas de réponse rapide de notre part, vous êtes en droit de faire parvenir vos doléances ou une demande de saisine au Défenseur des droits.
+
+## En savoir plus
```

Comment:
```suggestion
### En savoir plus
```

Reply:
Ok I will include "Report a malfunction" and "Learn More" as h3 in "Accessibilty" but this came from https://github.com/betagouv/juridiques/blob/main/Mentions-l%C3%A9gales.md

I did a PR to change that on their repo as I think it would be better https://github.com/betagouv/juridiques/pull/2